tests/tensor/test_math_opt.py

```diff
@@ -2742,26 +2742,19 @@ def setup_method(self):
         self.mode = (
             get_default_mode()
             .including("canonicalize", "fast_run")
-            .excluding("gpu", "fusion")
+            .excluding("gpu", "fusion", "inplace")
```

Comment:
Why is `"inplace"` added here?

Reply:
Makes the op checks easier to write. Otherwise we have to check against the inplace variations